### PR TITLE
[hi] localize continuous_integration.md

### PR DESCRIPTION
**Pull request by Aayush987**
Hi,
This Pr is Hindi localization of content/hi/continuous_integration.md page.

ref :  #270 
/lang hi

@anubha-v-ardhan  @Garima-Negi  and everyone plz review.

### CODE REVIEW

File: content/hi/continuous_integration.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: निरंतर इंटीग्रेशन
```
Comment (Garima-Negi) [inline content/hi/continuous_integration.md:2]: We can probably have `एकीकरण` in the title  for integration.

File: content/hi/continuous_integration.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है 
+
+निरंतर इंटीग्रेशन , जिसे  ज्यादातर  सी.आई. बोला जाता है, यह कोड परिवर्तन को  जल्दी से जल्दी  एकीकृत करने का एक अभ्यास है। सी.आई. [सी.डी](/Continuous_Delivery/) पढ़ने के लिए एक ज़रूरी अवधारणा है। ज़्यादातर सी.आई. की प्रक्रिया तब शरू होती है जब कोड परिवर्तन  को स्रोत नियंत्रण सिस्टम (गित, सबवर्सन, और  मरक्यूरिअल) में कमिट किया जाता है और फिर वह ख़तम होता है एक परीक्षण किया हुए उत्पाद की तरह जो की तैयार है सी.डी सिस्टम में जाने के लिए।
```
Comment (Garima-Negi) [inline content/hi/continuous_integration.md:9]: Let's have निरंतर एकीकरण to start with and continuous integration in parenthesis. As we are mentioning the abbreviation सी.आई. in the same sentence, it's ok to refer to that in remaining page.

File: content/hi/continuous_integration.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## समस्या
+
+ज़्यादातर सॉफ्टवेयर सिस्टम  बहुत  बड़े और जटिल होते है जिसको कई डेवेलपर्स रोज़ मेन्टेन और अपडेट करते है। वह लोग समानांतर अपने सिस्टम पर काम करते है जिस कारण कई कान्फ्लिक्टिंग परिवर्तन आ सकते है और अनजाने में एक दूसरे के कोड में  दिक्कत आ जाती  है। इसके साथ ही विभिन्न डेवेलपर्स रोज़ एक ही प्रोजेक्ट में काम करते है, तो जो रोजमर्रा के कार्य है जैसे कोड का परिक्षण और क्वालिटी की जाँच प्रत्येक डेवलपर द्वारा दोहराया जाना होगा जो की एक समय की बर्बादी होगी।
```
Comment (Garima-Negi) [inline content/hi/continuous_integration.md:13]: few suggestions here:
- conflicting can be localized to `विवादग्रस्त`
- क्वालिटी can be localized to `गुणवत्ता`

File: content/hi/continuous_integration.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+## समस्या
+
+ज़्यादातर सॉफ्टवेयर सिस्टम  बहुत  बड़े और जटिल होते है जिसको कई डेवेलपर्स रोज़ मेन्टेन और अपडेट करते है। वह लोग समानांतर अपने सिस्टम पर काम करते है जिस कारण कई कान्फ्लिक्टिंग परिवर्तन आ सकते है और अनजाने में एक दूसरे के कोड में  दिक्कत आ जाती  है। इसके साथ ही विभिन्न डेवेलपर्स रोज़ एक ही प्रोजेक्ट में काम करते है, तो जो रोजमर्रा के कार्य है जैसे कोड का परिक्षण और क्वालिटी की जाँच प्रत्येक डेवलपर द्वारा दोहराया जाना होगा जो की एक समय की बर्बादी होगी।  
+
+## समाधान 
+
+सी.आई. सॉफ्टवेयर खुद ब खुद जाँच करता है की कोड अच्छे से मर्ज हो जाए जब कभी भी कोई डेवलपर परिवर्तन कमिट करता है। यह एक लगभग सर्वव्यापी अभ्यास है  की हम सी.आई. सर्वर का उपयोग कोड की क्वालिटी की जाँच ,परिक्षण और यहां तक ​​कि डेप्लोय्मेंट्स के लिए भी करते है। इसी चलते यह टीमों के भीतर कोड परिक्षण के लिए एक ठोस कार्यान्वयन बनता है। सी.आई. सॉफ्टवेयर टीमों को अनुमति देता है की वह किसी भी कोड कमिट को या तो ठोस असफलता या फिर एक व्यवहार्य रिलीज उम्मीदवार में बदल सकता है।
```
Comment (Garima-Negi) [inline content/hi/continuous_integration.md:17]:
```suggestion
सी.आई. सॉफ्टवेयर स्वचालित रूप से जाँच करता है की कोड अच्छे से मर्ज हो जाए जब कभी भी कोई डेवलपर परिवर्तन कमिट करता है। यह एक लगभग सर्वव्यापी अभ्यास है  की हम सी.आई. सर्वर का उपयोग कोड की गुणवत्ता की जाँच, परिक्षण और यहां तक ​​कि डेप्लोय्मेंट्स के लिए भी करते है। यह टीमों के भीतर कोड परिक्षण के लिए एक ठोस कार्यान्वयन बन जाता है। सी.आई. सॉफ्टवेयर टीमों को अनुमति देता है की वे किसी भी कोड कमिट को या तो ठोस असफलता या फिर एक व्यवहार्य रिलीज उम्मीदवार में बदल बदल सकते हैं। 
```

**Review by Garima-Negi — CHANGES_REQUESTED** [review]
Hi there, submitted few suggestions and changes as part of my review.

File: content/hi/continuous_integration.md (hunk 1 above)
Comment (Aayush987) [reply content/hi/continuous_integration.md:2]: Ok thank you. I will do it

File: content/hi/continuous_integration.md (hunk 2 above)
Comment (Aayush987) [reply content/hi/continuous_integration.md:9]: @Garima-Negi 
Do i have to add continuous integration in parenthesis next to सी.आई.?

Comment (Aayush987) [reply content/hi/continuous_integration.md]: Done

File: content/hi/continuous_integration.md (hunk 3 above)
Comment (Aayush987) [reply content/hi/continuous_integration.md]: Done

File: content/hi/continuous_integration.md (hunk 4 above)
Comment (Aayush987) [reply content/hi/continuous_integration.md]: Done

**Review by Garima-Negi — APPROVED** [review]
LGTM

File: content/hi/continuous_integration.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: निरंतर एकीकरण
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है 
+
+निरंतर एकीकरण (Continuous_Integration) , जिसे  ज्यादातर  सी.आई. बोला जाता है, यह कोड परिवर्तन को जल्दी से जल्दी एकीकृत करने का एक अभ्यास है। सी.आई. [सी.डी](/Continuous_Delivery/) पढ़ने के लिए एक ज़रूरी अवधारणा है। ज़्यादातर सी.आई. की प्रक्रिया तब शरू होती है जब कोड परिवर्तन को स्रोत नियंत्रण सिस्टम (गित, सबवर्सन, और  मरक्यूरिअल) में कमिट किया जाता है और फिर वह ख़तम होता है एक परीक्षण किया हुए उत्पाद की तरह जो की तैयार है सी.डी सिस्टम में जाने के लिए।
```
Comment (anubha-v-ardhan) [inline content/hi/continuous_integration.md:9]:
```suggestion
निरंतर एकीकरण (Continuous_Integration) , जिसे  अक्सर CI कहा जाता है, कोड परिवर्तन को जल्दी से जल्दी एकीकृत करने का एक अभ्यास है। CI, [CD](/Continuous_Delivery/) पढ़ने के लिए आवश्यक है। ऐतिहासिक रूप से, CI की प्रक्रिया तब शरू होती है जब कोड परिवर्तन को सोर्स नियंत्रण सिस्टम (Git, Mercurial, या Subversal) में कमिट किया जाता है और वह परीक्षित उत्पाद, सी.डी सिस्टम में जाने के लिए तैयार होता है।
```

File: content/hi/continuous_integration.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+## समस्या
+
+ज़्यादातर सॉफ्टवेयर सिस्टम बहुत बड़े और जटिल होते है जिसको कई डेवेलपर्स रोज़ मेन्टेन और अपडेट करते है। वह लोग समानांतर अपने सिस्टम पर काम करते है जिस कारण कई विवादग्रस्त परिवर्तन आ सकते है और अनजाने में एक दूसरे के कोड में दिक्कत आ जाती है। इसके साथ ही विभिन्न डेवेलपर्स रोज़ एक ही प्रोजेक्ट में काम करते है, तो जो रोजमर्रा के कार्य है जैसे कोड का परिक्षण और गुणवत्ता की जाँच प्रत्येक डेवलपर द्वारा दोहराया जाना होगा जो की एक समय की बर्बादी होगी।
```
Comment (anubha-v-ardhan) [inline content/hi/continuous_integration.md:13]:
```suggestion
ज़्यादातर सॉफ्टवेयर सिस्टम बहुत बड़े और जटिल होते है जिसको कई डेवेलपर्स रोज़ मेन्टेन और अपडेट करते है। वह लोग समानांतर अपने सिस्टम पर काम करते है जिस कारण कई विवादग्रस्त परिवर्तन आ सकते है और अनजाने में एक दूसरे के कोड में दिक्कत आ जाती है। इसके साथ ही विभिन्न डेवेलपर्स रोज़ एक ही प्रोजेक्ट में काम करते है, जिससे रोजमर्रा के कार्य, जैसे कोड का परिक्षण और गुणवत्ता की जाँच प्रत्येक डेवलपर द्वारा दोहराया जाना पड़ता है जो समय की बर्बादी है।  
```

File: content/hi/continuous_integration.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+## समस्या
+
+ज़्यादातर सॉफ्टवेयर सिस्टम बहुत बड़े और जटिल होते है जिसको कई डेवेलपर्स रोज़ मेन्टेन और अपडेट करते है। वह लोग समानांतर अपने सिस्टम पर काम करते है जिस कारण कई विवादग्रस्त परिवर्तन आ सकते है और अनजाने में एक दूसरे के कोड में दिक्कत आ जाती है। इसके साथ ही विभिन्न डेवेलपर्स रोज़ एक ही प्रोजेक्ट में काम करते है, तो जो रोजमर्रा के कार्य है जैसे कोड का परिक्षण और गुणवत्ता की जाँच प्रत्येक डेवलपर द्वारा दोहराया जाना होगा जो की एक समय की बर्बादी होगी।  
+
+## समाधान 
+
+सी.आई. सॉफ्टवेयर स्वचालित रूप से जाँच करता है की कोड अच्छे से मर्ज हो जाए जब कभी भी कोई डेवलपर परिवर्तन कमिट करता है। यह एक लगभग सर्वव्यापी अभ्यास है की हम सी.आई. सर्वर का उपयोग कोड की गुणवत्ता की जाँच, परिक्षण और यहां तक ​​कि डेप्लोय्मेंट्स के लिए भी करते है। इसी चलते यह टीमों के भीतर कोड परिक्षण के लिए एक ठोस कार्यान्वयन बन जाता है। सी.आई. सॉफ्टवेयर टीमों को अनुमति देता है की वह किसी भी कोड कमिट को या तो ठोस असफलता या फिर एक व्यवहार्य रिलीज उम्मीदवार में बदल सकते हैं।
```
Comment (anubha-v-ardhan) [inline content/hi/continuous_integration.md:17]:
```suggestion
जब कभी भी कोई डेवलपर परिवर्तन कमिट करता है, CI सॉफ्टवेयर स्वचालित रूप से जाँच करता है की कोड अच्छे से मर्ज हो जाए। यह लगभग एक सर्वव्यापी अभ्यास है की हम CI सर्वर का उपयोग कोड की गुणवत्ता की जाँच, परिक्षण और यहां तक ​​कि डेप्लोय्मेंट्स के लिए भी करते है। इसी चलते यह टीमों के भीतर कोड परिक्षण के लिए एक ठोस कार्यान्वयन बन जाता है। CI सॉफ्टवेयर टीमों को अनुमति देता है की वह किसी भी कोड कमिट को या तो ठोस असफलता या फिर एक व्यवहार्य रिलीज में बदल सकते हैं। 
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you @Aayush987 
Please go through suggestion comments and commit them if you agree. Feel free to discuss if you find some inaccuracy or feel like a change is unnecessary.

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]
+1 to @anubha-v-ardhan 's suggestions. Kindly make the mentioned changes.

**Review by anubha-v-ardhan — APPROVED** [review]
Thank you @Aayush987 :)

**Comment by anubha-v-ardhan** [comment]
@jayesh-srivastava If you think this PR is ready you can give your approval and "squash and merge" this PR.
I would also suggest not using "request changes" for a simple comment and instead using "comment" option if you're suggesting no change. 
"request changes" blocks the PR from getting merged even when its ready. :)

File: content/hi/continuous_integration.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+---
+title: निरंतर एकीकरण
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है 
+
+निरंतर एकीकरण (Continuous_Integration) , जिसे  अक्सर CI कहा जाता है, कोड परिवर्तन को जल्दी से जल्दी एकीकृत करने का एक अभ्यास है। CI, [CD](/Continuous_Delivery/) पढ़ने के लिए आवश्यक है। ऐतिहासिक रूप से, CI की प्रक्रिया तब शरू होती है जब कोड परिवर्तन को सोर्स नियंत्रण सिस्टम (Git, Mercurial, या Subversal) में कमिट किया जाता है और वह परीक्षित उत्पाद, सी.डी सिस्टम में जाने के लिए तैयार होता है।
```
Comment (jayesh-srivastava) [inline content/hi/continuous_integration.md:9]: `सी.डी` here should be written as CD only. @anubha-v-ardhan thoughts?

Comment (anubha-v-ardhan) [reply content/hi/continuous_integration.md:9]: Ah! yes, I must have missed.

**Comment by Aayush987** [comment]
Closing this pr due to some error from my side.

**Pull request closed without merging**